packages/k8s-triliovault-operator/k8s-triliovault-operator.patch

```diff
@@ -0,0 +1,32 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/k8s-triliovault-operator/charts-original/Chart.yaml packages/k8s-triliovault-operator/charts/Chart.yaml
+--- packages/k8s-triliovault-operator/charts-original/Chart.yaml
++++ packages/k8s-triliovault-operator/charts/Chart.yaml
+@@ -1,6 +1,7 @@
+ apiVersion: v1
+ appVersion: v2.0.2
+-description: K8s-TrilioVault-Operator is an operator designed to manage the K8s-TrilioVault Application Lifecycle.
++description: K8s-TrilioVault-Operator is an operator designed to manage the K8s-TrilioVault
++  Application Lifecycle.
+ home: https://github.com/trilioData/k8s-triliovault-operator
+ icon: https://www.trilio.io/wp-content/uploads/2021/01/Trilio-2020-logo-RGB-gray-green.png
+ maintainers:
+@@ -10,3 +11,6 @@
+ sources:
+ - https://github.com/trilioData/k8s-triliovault-operator
+ version: v2.0.2
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: k8s-triliovault-operator
+diff -x '*.tgz' -x '*.lock' -uNr packages/k8s-triliovault-operator/charts-original/app-readme.md packages/k8s-triliovault-operator/charts/app-readme.md
+--- packages/k8s-triliovault-operator/charts-original/app-readme.md
++++ packages/k8s-triliovault-operator/charts/app-readme.md
+@@ -0,0 +1,4 @@
++# TrilioVault for Kubernetes
++
++[K8s-TrilioVault-Operator](https://trilio.io) is an operator designed to manage the K8s-TrilioVault Application 
++Lifecycle.
+diff -x '*.tgz' -x '*.lock' -uNr packages/k8s-triliovault-operator/charts-original/questions.yml packages/k8s-triliovault-operator/charts/questions.yml
+--- packages/k8s-triliovault-operator/charts-original/questions.yml
++++ packages/k8s-triliovault-operator/charts/questions.yml
+@@ -0,0 +1 @@
++questions:
```

Comment:
Same as `app-readme.md` (see other comment), shouldn't be in the patch.

Also, questions are used in the UI to provide the user with a nicer way to configure the chart. Please include at least a few questions to cover common use cases for your chart.

Reply:
We don't have any questios for users to answer for helm chart so if required user can update the `values.yaml` and I will remove the `questions.yaml` from the PR

Reply:
Questions populate the chart's values.yaml. It is just a way to configure it through the UI rather than modify the file itself. If you would rather have users modify the chart's values.yaml, you can delete the questions from the overlay directory, and make sure the patch doesn't contain any traces of it.

Reply:
I have removed the `questions.yaml` since user does not need to asnwer any questions and parameters present in the `values.yaml` and good for installation.